extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -732,14 +731,11 @@ export class ProjectsController {
 
 		let extractTargetOptions: QuickPickItem[] = [];
 
-		let keys: string[] = Object.keys(ExtractTarget).filter(k => typeof ExtractTarget[k as any] === 'number');
+		let keys: string[] = Object.values(ExtractTarget).filter(k => k !== 'dacpac'); //Do not present the option to create Dacpac
 
 		// TODO: Create a wrapper class to handle the mapping
 		keys.forEach((targetOption: string) => {
-			if (targetOption !== 'dacpac') {		//Do not present the option to create Dacpac
-				let pascalCaseTargetOption: string = utils.toPascalCase(targetOption);	// for better readability
```

Comment:
Can you also remove the util methods and the related test cases for it: toPascalCase, toCamelCase (if still present).

Reply:
removed :)